sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentType.md

```diff
@@ -33,6 +33,12 @@ Add-PnPContentType -Name "Project Document" -Description "Use for Contoso projec
 ```
 
 This will add a new content type based on the parent content type stored in the $ct variable.
+### EXAMPLE 2
+```powershell
+Add-PnPContentType -Name "Project Document" -Description "Use for Contoso projects" -Group "Contoso Content Types" -ParentContentType (Get-PnPContentType -Identity 0x0101) -DocumentTemplate "/_cts/Project Document/template.docx"
+```
+
+This will add a new content type based on the standard document content type and assigns the document template template.docx to it
```

Comment:
```suggestion
This will add a new content type based on the standard document content type and assigns the document template template.docx to it.
```